.travis.yml

```
@@ -39,7 +39,7 @@ jobs:
 
                 - echo "memory_limit=4096M" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
 
-                - composer self-update --1
+                - composer self-update --preview
```

Since it's stable already:

```suggestion
                - composer self-update --2
```